examples/step-81/doc/intro.dox

```diff
@@ -0,0 +1,450 @@
+around 2.5 ångstrom ($2.5\times10^{-10}\text{m}$). We will thus model
+graphene as a lower-dimensional interface $\Sigma$ imbedded into the
+computational domain $\Omega\subset\mathbb{R}^d$. More precisely, $\Sigma$
+is a two-dimensional sheet in three spatial dimensions, or a
+one-dimensional line in two spatial dimensions. The special electronic
+structure of graphene gives rise to a current density on the
+lower-dimensional interface that is modeled with an effective surface
+conductivity $\sigma^\Sigma$ obeying <a
+href="https://en.wikipedia.org/wiki/Ohm%27s_law">Ohm's Law</a>:
+@f[
+  \mathbf{J}^\Sigma=\sigma^\Sigma\,\mathbf{E}_T
+@f]
+in which $\mathbf{J}^\Sigma$ is the surface current density, $\mathbf{E}_T$
+denotes the tangential part of the electric field $\mathbf{E}$, and
+$\sigma^\Sigma$ is an appropriately chosen surface conductivity that will
+be discussed in more detail below. The surface current density gives rise
+to a jump condition on $\Sigma$ in the tangential component of the magnetic
+field. This is best seen by visualizing <a
+href="https://en.wikipedia.org/wiki/Amp%C3%A8re%27s_circuital_law">Ampère's
+law</a>:
+
+@htmlonly
+<p align="center">
+  <img src="https://www.dealii.org/images/steps/developer/step-81-cartoon.png" alt="Visualization of Ohm's law and Ampère's law leading to a jump condition over the interface" height="200">
+</p>
+@endhtmlonly
+
+and then taking the limit of the upper and lower part of the line integral
+approaching the sheet. In contrast, the tangential part of the electric
+field is continuous. By fixing a unit normal $\mathbf{\nu}$ on the hypersurface
+$\Sigma$ both jump conditions are
+@f{align*}
+\mathbf{\nu} \times \left[(\mu^{-1}\mathbf{H})^+ - (\mu^{-1}\mathbf{H})^-\right]|_{\Sigma}
+&= \sigma^{\Sigma}\left[(\mathbf{\nu}\times \mathbf{E}\times \mathbf{\nu})\right]|_{\Sigma},
+\\
+\mathbf{\nu} \times \left[\mathbf{E}^+ - \mathbf{E}^-\right]|_{\Sigma} &= 0.
+@f}
+The notation $\mathbf{F}^\pm$ indicates the limit values of the field
+when approaching the interface from above or below the interface:
+$\mathbf{F}^\pm(\mathbf{x})=\lim_{\delta\to0,\delta>0}\mathbf{F}(\mathbf{x}\pm\delta\mathbf{\nu})$.
+
+
+<h4> Rescaling </h4>
+
+We will be using a rescaled version of the Maxwell's equations described above.
+The rescaling has the following key differences:
+<ol>
+<li>Every length is rescaled by the free-space wavelength $2\pi k^{-1}
+\dealcoloneq 2\pi(\omega\sqrt{\varepsilon_0\mu_0})^{-1}$, in which $\varepsilon_0$ and $\mu_0$
+denote the vacuum dielectric permittivity and magnetic permeability, respectively.</li>
+<li>$\mathbf{E}$, $\mathbf{H}$, $\mathbf{J}_a$, $\mathbf{M}_a$ are all rescaled by
+typical electric current strength $J_0$, i.e., the strength of the
+prescribed dipole source at location $a$ in the $e_i$ direction in Cartesian
+coordinates (here, $\delta$ is the Dirac delta operator).
+@f[
+\mathbf{J}_a = J_0 e_i\delta(x-a)
```

Comment:
```suggestion
\mathbf{J}_a = J_0 \mathbf{e}_i\delta(x-a)
```
At least something should be bold face on the rhs as well. (Also for the use in the main file.)